### DISCUSSION TITLE
[n64] Question when dumping different save game types

### DISCUSSION THREAD

**Discussion by vgmoose** [sanni/cartreader#422] [Need help]
Hi! I found the following page that goes over the different N64 save game types and which cartridges use which: http://micro-64.com/database/gamesave.shtml

On the wiki, it says for N64 reading to flip the switches like this: https://github.com/sanni/cartreader/wiki/Read-N64-cartridge (left, right, left, left)

The second switch is for "eeprom", and on the v3 overview page it states that this switch "Enables a 1K pull-up resistor if set to the right (only needed for N64 eeprom and controller)."

Does that mean this switch should only be flipped to the right when dumping an eeprom game? Or should it always be flipped? For example, if you have a flashram save (like Pokemon stadium 2 or Majora's mask) or a battery save, does it matter which orientation it's in?

Thank you!

**Comment by sanni** [comment]
You can leave it on at all times.

**Reply by vgmoose** [reply]
Thanks very much!
